int32 drone_id_from

FreeSpaceMsg[] free_space_list

geometry_msgs/Point[] key_pose_list

int32 last_send_inx